### PR TITLE
visiting-card-assignment design no.77 done

### PR DESCRIPTION
**Pull request by raveeshrajput**

### CODE REVIEW

File: SubmittedAssignments/Raveesh_Rajput/preview.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                    <img class="orgLogo" src="./img/org-logo.png" alt="company-logo">
+                </div>
+                <div class="qr-code">
+                    <img src="./img/qr.png" alt="image-qr">
+                </div>
+                <div class="contact-info">
+                    <div class="phone-icon">
+                        <img class="phone-icon1" src="./icons/phone.png" alt="locationIcon">
+                    </div>
+                    <div class="phone">
```
Comment (akankshadube08) [inline SubmittedAssignments/Raveesh_Rajput/preview.html:35]: change the class name  from **phone** to **party-phone**

File: SubmittedAssignments/Raveesh_Rajput/preview.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                        <p>Phone: 6739793002</p>
+                    </div>
+                    <div class="email-icon">
+                        <img class="email-icon1" src="./icons/mail.png" alt="locationIcon">
+                    </div>
+                    <div class="email">
```
Comment (akankshadube08) [inline SubmittedAssignments/Raveesh_Rajput/preview.html:41]: change the class name from **email** to  **party-email**

**Review by akankshadube08 — CHANGES_REQUESTED** [review]
check the sample code again and make changes as per sample code to your code

**Review by akankshadube08 — CHANGES_REQUESTED** [review]
style.css should be in css folder

File: SubmittedAssignments/Raveesh_Rajput/preview.html (hunk 1 above)
Comment (raveeshrajput) [reply SubmittedAssignments/Raveesh_Rajput/preview.html]: changes updated.

**Comment by raveeshrajput** [comment]
please check my assignment.

**Review by akankshadube08 — APPROVED** [review]

**Pull request merged**